overrides.nix

```diff
@@ -3,9 +3,12 @@
 
   vadimcn.vscode-lldb = _: {
     postInstall = ''
```

Comment:
@szinn, you can use `lib.optionalString pkgs.stdenv.isLinux '' ... ''` to decide whether to provide a `postInstall` string rather than writing an `if-` condition in `bash`.

Reply:
Done and much cleaner